db/migrate/20181206142708_create_other_asset_declaration.rb

```diff
@@ -0,0 +1,22 @@
+class CreateOtherAssetDeclaration < ActiveRecord::Migration[5.2]
+  def change
+    create_table :other_assets_declarations, id: :uuid do |t|
+      t.uuid :legal_aid_application_id, null: false
+      t.decimal :second_home_value, precision: 14, scale: 2
+      t.decimal :second_home_mortgage, precision: 14, scale: 2
+      t.decimal :second_home_percentage, precision: 14, scale: 2
+      t.decimal :timeshare_value, precision: 14, scale: 2
+      t.decimal :land_value, precision: 14, scale: 2
+      t.decimal :jewellery_value, precision: 14, scale: 2
+      t.decimal :vehicle_value, precision: 14, scale: 2
+      t.decimal :classic_car_value, precision: 14, scale: 2
+      t.decimal :money_assets_value, precision: 14, scale: 2
+      t.decimal :money_owed_value, precision: 14, scale: 2
+      t.decimal :trust_value, precision: 14, scale: 2
```

Comment:
Is there any reason for the precision and scale values being specifically set as 14 and 2 respectively? Current `decimal` setting are using default values. If `precision: 14` is needed for these values, should the existing attributes also be updated?

Reply:
I thought that it would allow money values of more than 2 decimal places to be stored, but testing shows that the default is to truncate it to 2, so happy to remove that.